cli/flox/doc/flox-activate.md

```diff
@@ -29,7 +29,14 @@ interactive shell.
 Launches a subshell non-interactively when invoked with a command
 and arguments.
 May also be invoked as `$(flox activate)` to produce commands to be sourced
-by your current `$SHELL`.
+by the parent shell.
+The parent shell is detected automatically by inspecting the executable path
+of the parent process.
```

Comment:
nonblocking: I think inspecting the executable path is an implementation detail. We could instead decide to inspect env vars or something. So I think it's sufficient to say we try to detect the parent shell